Modules/ChequeModule/Views/ChequeView.xaml

```diff
@@ -17,17 +18,160 @@
-            <RowDefinition Height="*" />
+            <RowDefinition Height="17*" />
+            <RowDefinition Height="331*"/>
+            <RowDefinition Height="15*"/>
+            <RowDefinition Height="60*"/>
+            <RowDefinition Height="23*"/>
         </Grid.RowDefinitions>
         <Grid.ColumnDefinitions>
             <ColumnDefinition Width="auto" />
+            <ColumnDefinition Width="3*"/>
+            <ColumnDefinition Width="55*"/>
+            <ColumnDefinition Width="4*"/>
+            <ColumnDefinition Width="84*"/>
+            <ColumnDefinition Width="4*"/>
         </Grid.ColumnDefinitions>
         <Button
-            Grid.Column="0"
             Command="{Binding NavigateBack}"
-            Style="{StaticResource BackImageButtonStyle}" />
+            Style="{StaticResource BackImageButtonStyle}" HorizontalAlignment="Center" Width="50" />
+
+        <StackPanel Grid.Row="4" Grid.Column="2">
+            <Button 
+                Width="101" 
+                Height="39" 
+                HorizontalAlignment="Left" >Save</Button>
+        </StackPanel>
+        <StackPanel Grid.Row="4" Grid.Column="2">
+            <Button 
+                Width="101" 
+                Height="39" 
+                HorizontalAlignment="Right" >Cancel</Button>
+        </StackPanel>
+
+        <StackPanel Grid.Row="1" Grid.Column="2" Margin="0,15,0,0" Grid.RowSpan="2" >
+            <StackPanel>
+                <TextBlock
+                    Text="Payee's name:" 
+                    HorizontalAlignment="Left" 
+                    Style="{StaticResource UserLoginTitleTextStyle}"/>
+                <TextBox
+                    Style="{StaticResource CommonTextBoxStyle}"/>
+            </StackPanel>
+            <Border Height="8"/>
+
+            <StackPanel>
+                <TextBlock
+                    Text="Amount (numbers):"
+                    HorizontalAlignment="Left" 
+                    Style="{StaticResource UserLoginTitleTextStyle}"/>
+                <TextBox
+                    Style="{StaticResource CommonTextBoxStyle}"/>
+            </StackPanel>
+            <Border Height="8"/>
+
+            <StackPanel>
+                <TextBlock
+                    Text="Date (DDMMYY):" 
+                    HorizontalAlignment="Left" 
+                    Style="{StaticResource UserLoginTitleTextStyle}"/>
+                <TextBox
+                    Style="{StaticResource CommonTextBoxStyle}"/>
+            </StackPanel>
+            <Border Height="8"/>
+
+            <StackPanel>
+                <TextBlock
+                    Text="Drawer:" 
+                    HorizontalAlignment="Left" 
+                    Style="{StaticResource UserLoginTitleTextStyle}"/>
+                <TextBox
+                    Style="{StaticResource CommonTextBoxStyle}"/>
+            </StackPanel>
+        </StackPanel>
+
+        <StackPanel Grid.Row="1" Grid.Column="4" Margin="0,15,0,0" Grid.RowSpan="2" >
```

Comment:
The Line 97 to Line 174 should be displaying like image below?
![image](https://github.com/kevinboon3288/cheque-writer/assets/38966644/911d79e3-2ab7-46ab-8ddc-e501c4320f51)

If so, there is no item displayed when I ran the application. (Tried to enlarge the resolution, still no item displayed)
![image](https://github.com/kevinboon3288/cheque-writer/assets/38966644/b45384fd-2b45-4186-b8c1-1e1229d9f10c)